jsonschema-hyperschema.xml

```diff
@@ -162,180 +158,129 @@
-                        a "collection" link, or on discovering an "item" link which has the
-                        instance or sub-instance as its context.
-                    </t>
-                </section>
-                <section title="Providing access to target attributes and input schemas">
-                    <t>
-                        For a given link, an implementation MUST make the values of all
-                        target attribute keywords directly available to the client.
-                        Implementations MAY provide additional interfaces for using this
-                        information, which is discussed in each keyword's section.
-                    </t>
-                    <t>
-                        For a given link, an implementation MUST make the value of each
-                        input schema keyword directly available to the client.
-                    </t>
-                </section>
-            </section>
-            <section title="Additional functionality">
                 <t>
-                    Implementations MAY provide additional functionality coordinating the use of
-                    JSON Hyper-Schema with other specifications, including but not limited to
-                    JSON Schema validation, serializing payloads based on media types, and
-                    communicating with resources via protocols as indicated by URI schemes.
-                    <cref>
-                        This is vague.  I'm trying to convey that most uses of Hyper-Schema
-                        would benefit from including integration with the media types, URI schemes,
-                        and protocols, such as automatically validating submission data against
-                        "submissionSchema", encoding it according to "submissionMediaType",
-                        examining the URI for the "http" or "https" scheme, validating JSON
-                        representations of headers against "headerSchema", serializing them
-                        into HTTP header format, and constructing the appropriate POST request.
-                        While many of those activities are technically outside of the scope of
-                        Hyper-Schema, coordinating all of them is specific to Hyper-Schema and
-                        beneficial to automate.
-                    </cref>
+                    This specification defines the following terms:
+                    <list style="hanging">
+                        <t hangText="JSON Hyper-Schema">
+                            A JSON Schema using the keywords defined by this specification.
+                        </t>
+                        <t hangText="hyper-schema">
+                            Within this document, the term "hyper-schema" always refers to
+                            a JSON Hyper-Schema
+                        </t>
```

Comment:
Information in the introduction is redundant almost by definition.  I think it's fundamental enough to stay in the introduction, and it's also necessary to include them in this definition list for completeness.  Otherwise it would be an odd omission if you just jumped into the definitions section without paying much attention to introduction.

My view of the introduction is that it should hit the major high points but not be essential.

I'm open to further debate on this, but unless you feel strongly I'd like to leave this as-is.

Reply:
Not a big deal, let's leave it as is.